doc/stdenv.xml

```diff
@@ -318,7 +318,13 @@ containing some shell commands to be executed, or by redefining the
 shell function
 <varname><replaceable>name</replaceable>Phase</varname>.  The former
 is convenient to override a phase from the derivation, while the
-latter is convenient from a build script.</para>
+latter is convenient from a build script.
+
+However, typically one only wants to <emphasis>add</emphasis> some
```

Comment:
Well, yes, but what if I really want to _replace_ the default phase behavior, but keep the hooks. We should really explain that.